code/modules/mob/living/carbon/monkey/life.dm

```diff
@@ -12,11 +12,18 @@
 	if(..())
 
 		if(!client)
-			if(stat == CONSCIOUS)
-				if(!handle_combat())
-					if(prob(33) && canmove && isturf(loc) && !pulledby)
+			if(stat == CONSCIOUS)	
+				if(on_fire || buckled || restrained())

```

Comment:
add check for (resting && canmove) so that AI monkeys attempt to resist if they're resting

Reply:
That was probably a source of some of our issues. why would we want this?

Reply:
Adding a (resting && canmove) ensures monkeys actually try to stand up after getting knocked down, instead of continuing to mindlessly fight on the ground without even trying to stand up.

AI monkeys dont have autostand. adding the (resting && canmove) check is necessary so monkeys don't stay on the ground for the rest of their lives when they get pushed over.